osu.Game/Rulesets/UI/RulesetContainer.cs

```diff
@@ -43,7 +42,7 @@ public abstract class RulesetContainer : Container
         /// <summary>
         /// The input manager for this RulesetContainer.
         /// </summary>
-        internal readonly PlayerInputManager InputManager = new PlayerInputManager();
+        internal IHasReplayHandler ReplayInputManager => (IHasReplayHandler)KeyBindingInputManager;

```

Comment:
(for what it's worth i'd rather this exception is fired, since it means the ruleset is accepting replays but hasn't implemented the ability to do so... or to even use actions at that).